### DISCUSSION TITLE
Does anyone have a screenshot of this operating system?

### DISCUSSION THREAD

**Discussion by ghost** [CosmosOS/Cosmos#2405] [Q&A]
Hi! I would like to see how it is, before installing but I can't find any information about it. Does anyone have a screenshot of this operating system?

**Comment by BlitzWolf007** [comment]
What operating system? Cosmos is a tool to create operating systems, not an operating system itself.

**Reply by ghost** [reply]
I thought it was an operating system operating, anyway thank you for feedback! sorry for the inconvenience.
